### DISCUSSION TITLE
Are there autoplay and loop settings for the Youtube extension?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#770] [Ask a question] [answered]
Hi,

I was wondering if it is possible to get the [Youtube extension](https://github.com/datenstrom/yellow-extensions/tree/master/source/youtube) to autoplay the video and to loop it. It would also be nice to be able to remove all Youtube frame decorations and buttons, so that it's just a moving picture on the screen with no controls.

From what I found on the Internet, it seems that the desired parameters still supported as of 2022 are:

```
controls=0
modestbranding=1
autoplay=1
loop=1
```

But as I try to add them in the `yellow-system.ini` as `YoutubeStyle: flexible controls=0 modestbranding=1 autoplay=1 loop=1` or inside a Markdown file as `[youtube 12345abcdef&controls=0;modestbranding=1;autoplay=1;loop=1]` - they don't work.

Tip: You can edit your own questions here on GitHub, click on `...` -> `Edit`.

**Comment by schulle4u** [comment]
Sorry, the Youtube extension currently doesn't support additional parameters other than the options available in the documentation. 
You could try to copy/modify  the embed code from Youtubes sharing window, but this might cause problems with Yellow's HTML filter.

**Comment by schulle4u** [comment, marked as answer]
Here is an example for embedding Youtube videos with autoplay and looping enabled using HTML. 

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/fhs55HEl-Gc?controls=0&modestbranding=1&autoplay=1&loop=1" frameborder="0" allow="autoplay; fullscreen"></iframe>
```

**Reply by glow12121** [reply]
Thank you. It works - no control buttons, and autoplay.

Is there an easy way to apply Yellow's CSS for Youtube and make the video responsive?

**Reply by glow12121** [reply]
I have found a workaround. Vimeo has a simpler way to embed videos than Youtube. To whom it may concern - it is important though to include `muted=1` parameter. Otherwise the video won't autoplay on more modern browsers.

```
<div style="padding:75% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/101010101?h=ab1010abab&amp;badge=0&amp;autoplay=1&amp;loop=1&amp;autopause=0&amp;muted=1&amp;player_id=0&amp;app_id=12345" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen style="position:absolute;top:0;left:0;width:100%;height:100%;" title="video_2022-03-27_14-13-36.mp4"></iframe></div><script src="https://player.vimeo.com/api/player.js"></script>
```
